src/twisted/python/_setup.py

```diff
@@ -77,6 +77,7 @@
     'twisted-dev-tools >= 0.0.2',
     'python-subunit',
     'sphinx >= 1.3.1',
+    'hypothesis >= 3.6.1'
```

Comment:
This will need to be bumped to take advantage of hopefully merged invariant support upstream.